### ISSUE TITLE
Show number input on experiment collection dashboard that allows the researcher to navigate to a specific participant's dashboard

### ISSUE THREAD

**Issue by drikusroor** [Amsterdam-Music-Lab/MUSCLE#914]
**Original title of this Issue:** "Show number input on experiment collection dashboard that allows the researcher to navigate to a specific participant's dashboard"

**Is your feature request related to a problem? Please describe.**

Related to and blocked by #913, the experiment collection dashboard will show the *started* and *finished* count number for the experiments for a specific participant when the participant id query parameter has been added to the url.

On the Thoughbook, however, it's fairly difficult to open the address bar and add the `?participant_id=123` query parameter to the current url. It might therefore be convenient ~to add a number input to the dashboard~ have a solution that allows the researcher to directly ~input the participant id~ navigate to a participant's dashboard and don't have to worry about the right syntax and the address bar.

**Describe the solution you'd like**

- On the top of the dashboard will be a number input and a submit button
- The user enters the participant id into the number input and submits the number
- The application will then navigate to the same experiment collection page, but now with the right query parameter added to the url 

![image](https://github.com/Amsterdam-Music-Lab/MUSCLE/assets/8208970/d12a8bae-a9d7-49d0-8c2e-d9c821d970da)

**Comment by BeritJanssen** [comment]
I repeat my recommendation for this particular problem: make bookmarks ahead of time for the participant ids. I cannot foresee this functionality being used in other experiments. @Lia-JX-Li can you try the bookmark approach first?

**Comment by BeritJanssen** [comment]
We discussed that we might make a losse html file with Javascript which has just an input field, and then opens `localhost:3000/collection/name-of-collection?participant_id=someid` in a new tab. If we want to version control this, it could be in its own repo (as I don't think we'll integrate this into MUSCLE proper). This will be something that can be achieved relatively quickly, but it's a nice to have.

**Comment by drikusroor** [comment]
> We discussed that we might make a loose html file with Javascript which has just an input field, and then opens `localhost:3000/collection/name-of-collection?participant_id=someid` in a new tab. If we want to version control this, it could be in its own repo (as I don't think we'll integrate this into MUSCLE proper). This will be something that can be achieved relatively quickly, but it's a nice to have.

Good point. I've updated the PR to propose a "solution" to the problem instead of the specific technical solution that was proposed before.

**Comment by BeritJanssen** [comment]
I still thought of another solution: we might add a button "launch experiment with this participant" from the admin interface. But the design choices involved in that make this a more involved solution. But I think in the long run, functionality along these lines should be available through admin, not through the participant-facing frontend.

**Comment by BeritJanssen** [comment]
We may revisit this after #931 .

**Comment by BeritJanssen** [comment]
After the departure of the Congo team, I think we don't need this issue anymore, though I can imagine we build something like this in as part of #979 .
